### DISCUSSION TITLE
Auxilio na criação de contrato para patrocínio / PyLadies Recife

### DISCUSSION THREAD

**Discussion by naanadr** [apyb/comunidade#85] [Pergunta]
Olá pessoal! 

Faço parte do PyLadies Recife e estamos com um [plano de patrocínio](https://recife.pyladies.com/plano_patrocinio_pyladiesrecife.pdf) e gostaríamos do auxilio da APyB para gerar um contrato de patrocínio. 

Vocês poderiam compartilhar o modelo de contrato de patrocínio utilizado na Python Nordeste e Python Brasil (recife[at]pyladies.com)? Vocês geralmente reconhecem firma, ou apenas precisa assinar utilizando o gov.br ?

**Comment by tiidadavena** [comment]
Antes de ceder o contrato, precisamos entender alguns pontos: se o valor de apoio da empresa madrinha entraria no caixa da APyB e também quem seria a(s) representante(s) da comunidade para a Diretoria entrar em contato.

**Reply by naanadr** [reply]
Atualmente estamos utilizando uma conta no PagBank em nome de uma pessoa física da comunidade, e disponibilizamos o extrato da conta para consulta da comunidade. 

No entanto, tenho algumas dúvidas:

1. Se descidíssimo colocar o dinheiro na caixa da APyB, seria muito complexo para gerenciar?  
2. Como poderíamos utilizar o dinheiro se estivesse na caixa da APyB? Pretendemos utilizar o dinheiro para pagar passagem, coffee break e outros custos que possam surgir.

**Reply by naanadr** [reply]
Olá @tiidadavena estamos no aguardo 😄

**Comment by tiidadavena** [comment]
Sobre a pergunta da assinatura, são usadas quaisquer ferramentas de assinatura digital do mercado (sei que já foram assinados documentos com a DocSign). Outra opção é a Autentique.
